CONTRIBUTOR-DOCS/01_contributor-guides/02_using-the-issue-tracker.md

```diff
@@ -0,0 +1,69 @@
+<!-- Generated breadcrumbs - DO NOT EDIT -->
+
+[CONTRIBUTOR-DOCS](../README.md) / [Contributor guides](README.md) / Using the issue tracker
+
+<!-- Document title (editable) -->
+
+# Using the issue tracker
+
+<!-- Generated TOC - DO NOT EDIT -->
+
+<details open>
+<summary><strong>In this doc</strong></summary>
+
+- [Before creating a new issue](#before-creating-a-new-issue)
+- [Filing a bug report](#filing-a-bug-report)
+- [Classifying issue severity](#classifying-issue-severity)
+- [Requesting a feature or a new component](#requesting-a-feature-or-a-new-component)
+
+</details>
+
+<!-- Document content (editable) -->
+
+We use GitHub Issues for two purposes:
+
+1. Bug Reports
+2. Feature Requests (after initial discussion)
+
+If you're having a usage issue or need support, do not open an issue. Instead, reference the Community & Support section. This helps us keep issues focused on actual bugs and actionable tasks.
+
+## Before creating a new issue
+
+1. Check the [Open Issues](https://github.com/adobe/spectrum-web-components/issues) to see if the problem has already been reported.
+    - TIP: Apply the component label to make your search process more straightforward.
+2. If it has and the issue is still open, add a comment to the existing issue instead of opening a new one.
+3. Check if you can reproduce the problem in the latest version of Spectrum Web Components.
+4. If there are no related open issues and it is reproducible in isolation, then open a [Bug Report](https://github.com/adobe/spectrum-web-components/issues/new?template=bug_report.yaml).
+
+## Filing a bug report
+
+When you file a bug, please use the `Bug Report` template provided in GitHub. Include the following information:
+
+- A concise summary of the problem.
+- Relevant components involved in the issue.
+- Issue Severity based on our classifications defined below.
+- What you expected vs. what actually happened, along with any errors logged in the console.
+- Steps to reproduce the issue, preferably in an isolated environment, so that we can narrow down where the bug is originating from. (e.g., webcomponents.dev or CodePen). Be detailed if you write out the steps!
```

Comment:
```suggestion
- Steps to reproduce the issue, preferably in an isolated environment, so that we can narrow down where the bug is originating from, preferably by forking an example from our [Stackblitz project](https://stackblitz.com/orgs/custom/SWC-Team/collections/spectrum-web-components). Be detailed if you write out the steps!
```